_posts/2017-12-28-Secure-your-architecture-part2.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: post
+authors: [kevin_van_houtte]
+title: 'Securing your cloud-native microservice architecture in Spring: part 2'
+image: /img/microservices/part2/part2logo.jpg
+tags: [Microservices, Security, Encryption, Spring, Cloud, Config]
+category: Microservices
+comments: true
+---
+
+Since the rise of the digital era, most enterprises keep their data in a digital format.
+But if their sensitive data lacks security, it can cause the data to be unreliable, unstable and unavailable to their business.
+We have to be prepared if an attacker breaches in to our network and tries to hack our sensitive data.
+Whether it is in-motion or at-rest, encrypting our data and using the proper protection mechanism, will make it worthless for the hacker to use. 
+
+# Overview
+* [Cryptographic Algorithms](#cryptographic-algorithms)
+* [Key Mechanics](#key-mechanics)
+* [Cloud-hosted Key management service](#cloud-hosted-key-management-service)
+* [Spring Cloud Config Server](#spring-cloud-config-server)
+
+# Cryptographic Algorithms
+When implementing our application, every programming language will provide us with a set of known libraries for cryptographic algorithms. 
+A big flaw is implementing an algorithm by yourself, the known algorithms have been reviewed, patched and been known for their excellent security. 
+These are the most used types that you can use for encryption at rest: 
+
+## Symmetric Encryption
```

Comment:
There doesn't appear to be a big styling difference between the `#` and `##` heading. Perhaps try some different styling to make it more clear that "Symmetric and Asymmetric Encryption" are part of "Cryptographic Algorithms"? The same applies to the following sections.